### DISCUSSION TITLE
Error When trying to use CivitAI Browser Plus

### DISCUSSION THREAD

**Discussion by Johnnybeaner** [BlafKing/sd-civitai-browser-plus#232] [General]
Hi;
I get the following error while trying to use the extension.
I tried remove the extenstion and reloading it and it still happens.
Any ideas?

![Screenshot 2024-03-16 113058](https://github.com/BlafKing/sd-civitai-browser-plus/assets/99362559/abfd94fb-c04a-41f7-a1c7-09078dc65b4f)

**Comment by BlafKing** [comment]
This is caused by a recent update in the CivitAI API,
I'm currently working on a fix! You can see more info about the issue here: #229
